dotnet/src/Connectors/Connectors.Memory.Qdrant/Http/ApiSchema/SearchVectorsResponse.cs

```diff
@@ -1,34 +1,32 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
-using System;
 using System.Collections.Generic;
 using System.Text.Json.Serialization;
 
-namespace Microsoft.SemanticKernel.Skills.Memory.Qdrant.Http.ApiSchema;
+namespace Microsoft.SemanticKernel.Connectors.Memory.Qdrant.Http.ApiSchema;
 
 #pragma warning disable CA1812 // Avoid uninstantiated internal classes: Used for Json Deserialization
-internal class SearchVectorsResponse<TEmbedding> : QdrantResponse
-    where TEmbedding : unmanaged
+internal class SearchVectorsResponse : QdrantResponse
 {
     internal class ScoredPoint
     {
         [JsonPropertyName("id")]
-        public string Id { get; set; }
+        public string Id { get; }
```

Comment:
Does this deserialize properly despite the removal of the "set"?

Consider using the "record" keyword instead of a class and the "init" keyword instead of a set.

Reply:
@glahaye this does deserialize properly. It's not covered in a unit test, but can be demonstrated with Example19.

I'm not really sure what your other comment is referring to.

Reply:
If it works, then don't worry about the second part of my comment.  It's referring to:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record

It's a nifty tool to have on your belt, especially for immutable data types.

Resolving this conversation.